features/profile/ProfileTagInput.tsx

```diff
@@ -195,9 +203,13 @@ export default function ProfileTagInput({
         className={classes.popper}
       >
         <div className={classes.header}>
-          <Typography>Press 'Enter' to add</Typography>
-          <Link href="mailto:support@couchers.org">Email us</Link> to add a
-          language
+          <Typography>
+            <Trans i18nKey="profile:profile_tag_input.header_text">
+              Press 'Enter' to add
+              <Link href="mailto:support@couchers.org">Email us</Link> to add a
+              language
+            </Trans>
```

Comment:
With `i18n`, I found it a bit annoying to need to write the same copy both in the locales file and here. I was playing with this for a personal project recently, and I discovered that passing the children isn't necessary. One can just define the copy in the locale file, and then use a `components` prop to map the referenced components in the copy to real components.

Sharing in case it helps keep these `<Trans>` usages simpler!

* Using `<Trans>`: https://github.com/guillermodlpa/deportravel-squeeze-page/blob/main/components/sections/Hero/Hero.tsx#L91
* Locale file: https://github.com/guillermodlpa/deportravel-squeeze-page/blob/main/public/locales/en/hero.json#L4

Reply:
Interesting, that does look a bit neater and also avoid having to sync the text in both places! Maybe I'll make the change here to get the ball rolling, and gradually update the other places (under "scout's rules" when changing wouldn't bloat the PR out of proportion) when proper feature development resumes